consensus/hotstuff_replica.go

```diff
@@ -25,45 +25,52 @@ var (
 
 func (handler *HotstuffReplicaMessageHandler) HandleNewRoundMessage(m *ConsensusModule, msg *typesCons.HotstuffMessage) {
 	handler.emitTelemetryEvent(m, msg)
+	defer m.paceMaker.RestartTimer()
```

Comment:
I feel a bit battled about inverting the order of these two lines (and below as well).

Reply:
Don't hesitate to bring stuff like this up. As a reader, you have a completely different view on it and this sort of things pays dividends.

If I had an explicit decision as to why I had a specific order, I'd push back, but I like this order better as well.

Plus, it took 10 seconds:

<img width="563" alt="Screen Shot 2022-09-28 at 5 43 11 PM" src="https://user-images.githubusercontent.com/1892194/192913205-6bf84f93-5680-4119-bec6-2aebc39e5aec.png">